targets/TARGET_TT/TARGET_TT_M4G9/Periph_Driver/inc/adc_ch.h

```diff
@@ -0,0 +1,135 @@
+/** 
+ *******************************************************************************
+ * @file    adc_ch.h
+ * @brief   This file provides all the functions prototypes for ADC driver. \n
+ *          Channel Class.
+ * @version V1.0.0.0
+ * $Date:: 2017-09-07 13:52:12 #$
+ * 
+ * DO NOT USE THIS SOFTWARE WITHOUT THE SOFTWARE LICENSE AGREEMENT.
+ * 
+ * (C)Copyright TOSHIBA MICROELECTRONICS CORPORATION 2017 All rights reserved
```

Comment:
what license is this released with ? Missing here

Reply:
It's doesn't matter.

Reply:
There is no limit for using this software，Please ignore this comment.

Reply:
This should be licensed under apache 2.0 as contribution to Mbed OS. 
As it is now it is confusing (there's no software license agreement).

Can you please update?